editor/assets/stylesheets/_z-index.scss

```diff
@@ -13,8 +13,8 @@ $z-layers: (
 	'.editor-post-visibility__dialog': 30,
 	'.editor-post-schedule__dialog': 30,
 	'.editor-block-mover': 30,
-	'.components-notice-list': 100000,
-	'.components-popover': 100000,
+	'.components-notice-list': 9989,
+	'.components-popover': 9989,
```

Comment:
Why `9989`?  How high does this value actually need to be in order to make it work, and why?  Let's avoid unnecessarily high values here, and if a specific value is needed, add a comment describing why.

See previous discussion about `z-index` values on https://github.com/WordPress/gutenberg/pull/637 - these are more complicated than many people realize.